dask_sql/physical/rel/logical/aggregate.py

```diff
@@ -513,11 +519,21 @@ def _perform_aggregation(
 
             aggregations_dict[input_col][output_col] = aggregation_f
 
+        group_columns = [
```

Comment:
Can't comment below but this is duplicated below and can probably be removed from line 542